shared/javascripts/tooltip.js

```diff
@@ -49,18 +49,18 @@ var privlyTooltip = {
      *
      * @param {string} dataDomain The domain controlling the content.
      * @param {string} newMessage The message to change the tooltip to. A
-     * limited set of characters are accepted: 
+     * limited set of characters are accepted:
      * digits, word characters, underscores (\w) and whitespace (\s), periods,
      * and colons.
      *
      */
     updateMessage: function(dataDomain, newMessage){
 
-      privlyTooltip.tooltipMessage = privlyTooltip.appName
-        + ":\u00A0" // non-breaking space
-        + newMessage
-        + ", from "
-        + dataDomain;
+      privlyTooltip.tooltipMessage = privlyTooltip.appName +
+        ":\u00A0" +    // non-breaking space
+        newMessage +
+        ", from " +
+        dataDomain;
```

Comment:
This is also (mostly) a matter of style that is preferred by some projects in my experience, but if you disagree, I can again revert the change from this PR.